crates/metrics/src/main.rs

```diff
@@ -31,35 +31,56 @@ enum Command {
         about = "Prints descriptive stats in csv format over a time range in cloudwatch"
     )]
     PrintCloudwatchStats {
+        #[structopt(flatten)]
+        cloudwatch_options: CloudwatchLogsOptions,
+
         #[structopt(
-            name = "region",
-            short = "r",
-            help = "The AWS region, defaults to eu-central-1."
-        )]
-        region: Option<Region>,
-        #[structopt(
-            name = "log_group_name",
-            short = "l",
-            help = "The AWS log group name to query over."
+            name = "aggregation_pattern",
+            long = "aggregation_pattern",
+            short = "g"
         )]
-        log_group_name: Option<String>,
+        aggregation_pattern: Option<String>,
+    },
+    #[structopt(
+        name = "print-cloudwatch-metrics",
+        about = "Prints the metrics for a cloudwatch query in csv format"
+    )]
+    PrintCloudwatchMetrics(CloudwatchLogsOptions),
+    #[structopt(
+        name = "print-log-metrics",
+        about = "Prints metrics in csv format over a log file"
+    )]
+    PrintLogMetrics {
+        #[structopt(name = "log_file", short = "f")]
+        log_file: PathBuf,
+    },
+    #[structopt(
+        name = "print-metric-stats",
+        about = "Prints descriptive stats in csv format over a metric csv file"
+    )]
+    PrintMetricStats {
+        #[structopt(name = "csv_file", short = "f")]
+        csv_file: PathBuf,
         #[structopt(
-            name = "assume_role_arn",
-            short = "a",
-            help = "Optional override for the amazon role to assume when querying"
+            name = "aggregation_pattern",
+            long = "aggregation_pattern",
```

Comment:
```suggestion
            long = "aggregation-pattern",
```

I think this would be a more idiomatic cli flag in kebab-case